src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/PXOverrideMismatch/Sources/PxOverrideInADifferentType.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+
+namespace Acuminator.Tests.Sources
+{
+	public class BaseClass : PX.Data.PXGraphExtension<MyGraph>
+	{
+		public virtual object TestMethod(int x, bool drilldown, double y)
+		{
+			return new object();
+		}
+	}
+
+	public class ExtClass : BaseClass
+	{
+		[PX.Data.PXOverride]
+		public override object TestMethod(int x, bool drilldown, double y)
+		{
+			return new object();
+		}
+	}
+
+	public class MyGraph : PX.Data.PXGraph
```

Comment:
```suggestion
	public class MyGraph : PX.Data.PXGraph<MyGraph>
```

Reply:
The last commit contains a code cleanup in the tests: naming, removal of the redundant PX.Data namespace, and PXGraph<TGraph>. Please let me know if it looks good.